app/assets/stylesheets/styles/components/users/_profile.css.scss

```diff
@@ -0,0 +1,136 @@
+.member-profile {
+
+  ul {
+    list-style: none;
+    margin-left: 0;
+  }
+
+  .member-content {
+    margin-top: 2rem;
+  }
+
+  .edit-profile-button {
+    margin-bottom: 1rem;
+    margin-right: 1rem;
+  }
+
+  .create-guide-button a {
+    color: #ffffff;
+  }
+
+  .profile-sub-header {
+    ul {
+      text-align: center;
+    }
+
+    li {
+      display: inline-block;
+      padding: 0.4rem 1rem;
+      margin: 0.4rem 0;
+      background-color: #e9e9e9;
+    }
+
+    .number {
+      font-weight: bold;
+      font-size: 1.2rem;
+      margin: 0;
+    }
+
+    .attribute {
+      color: #999999;
+      margin-bottom: 0.4rem;
+      font-weight: bold;
+    }
+  }
+
+  .sidebar-section, .section {
+    margin-bottom: 3rem;
+  }
+
+  .section-title {
+    display: block;
+    float: left;
+  }
+
+  .section-header {
+    border-bottom: 2px solid #cecece;
```

Comment:
Color literals like `#cecece` should only be used in variable declarations; they should be referred to via variable everywhere else.